funnel/views/project.py

```diff
@@ -167,6 +170,20 @@ def edit(self):
                 else:
                     return render_form(form=form, title=_("Edit project"), submit=_("Save changes"), autosave=True)
 
+    @route('cfp', methods=['GET', 'POST'])
+    @lastuser.requires_login
+    @requires_permission('edit_project')
+    def cfp(self):
+        form = CFPForm(obj=self.obj, parent=self.obj.profile, model=Project)
+        if form.validate_on_submit():
+            form.populate_obj(self.obj)
+            if self.obj.cfp_state.NONE:
```

Comment:
Only open the CfP if the user chose to open. It is possible for the user to save a private draft (cfp_state.NONE + non-blank instructions field).